src/main/kotlin/app/revanced/util/patch/MethodCall.kt

```diff
@@ -0,0 +1,48 @@
+package app.revanced.util.patch
+
+import app.revanced.patcher.extensions.replaceInstruction
+import app.revanced.patcher.util.proxy.mutableTypes.MutableMethod
+import org.jf.dexlib2.iface.instruction.formats.Instruction35c
+import org.jf.dexlib2.iface.reference.MethodReference
+
+interface IMethodCall {
```

Comment:
This interface should be package private "internal".

Reply:
Done.